api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -68,13 +73,22 @@ private String convertClusterName(String workspaceId) {
 
   @Autowired
   ClusterController(NotebooksService notebooksService,
-      Provider<User> userProvider) {
+      Provider<User> userProvider,
+      WorkspaceService workspaceService) {
     this.notebooksService = notebooksService;
     this.userProvider = userProvider;
+    this.workspaceService = workspaceService;
   }
 
   public ResponseEntity<Cluster> createCluster(String workspaceNamespace,
       String workspaceId) {
+
+    // This also enforces registered auth domain.
+    WorkspaceAccessLevel accessLevel;
+    accessLevel = workspaceService.enforceWorkspaceAccessLevel(workspaceNamespace,
```

Comment:
You don't use accessLevel anywhere. Just call enforceWorkspaceAccessLevel() here and don't capture the result?